yarn-project/archiver/src/archiver/archiver.ts

```diff
@@ -204,8 +208,14 @@ export class Archiver extends EventEmitter implements ArchiveSource, Traceable {
   private async syncSafe(initialRun: boolean) {
     try {
       await this.sync(initialRun);
+      return true;
     } catch (error) {
-      this.log.error('Error during sync', { error });
+      if (error instanceof NoBlobBodiesFoundError) {
+        this.log.error(`Error syncing archiver: ${error.message}`);
+      } else {
+        this.log.error('Error during archiver sync', error);
+      }
```

Comment:
Great that we have retriable errors now! Just wondering if it would make sense for the `else` branch to bubble the error up/crash the process?